src/Raven.Client/Documents/Session/SessionDocumentTimeSeries.cs

```diff
@@ -35,11 +35,21 @@ public void Append(DateTime timestamp, double value, string tag = null)
             _asyncSessionTimeSeries.Append(timestamp, value, tag);
         }
 
+        public void Append(TValues value)
+        {
+            _asyncSessionTimeSeries.Append(value.Timestamp, value.Values, value.Tag);
+        }
+
         public IEnumerable<TimeSeriesEntry> Get(DateTime? from = null, DateTime? to = null, int start = 0, int pageSize = int.MaxValue)
         {
             return AsyncHelpers.RunSync(() => _asyncSessionTimeSeries.GetAsync(from, to, start, pageSize));
         }
 
+        IEnumerable<TValues> ISessionDocumentTypedTimeSeries<TValues>.Get(DateTime? from, DateTime? to, int start, int pageSize)
+        {
+            return AsyncHelpers.RunSync(() => _asyncSessionTimeSeries.GetAsyncInternal<TValues>(from, to, start, pageSize));
```

Comment:
Shouldn't this call `GetAsync` not `GetAsyncInternal` ?

Reply:
`GetAsync` is only for `TimeSeriesEntry`, here you have `TValues`